doc/add_loop_library.rst

```diff
@@ -0,0 +1,227 @@
+===============
+Add new library
+===============
+
+FlexibleSUSY provides a loop library interface, which allows to setup several
+loop libraries once and then choose any of them at run time.
+
+If one has a wish to add another library to a set of existing ones then this
+document can be used in order to simplify required operations.
+
+Main steps and definitions
+``````````````````````````
+
+All procedure can be divided into the following set:
+
+* Create c++ code of the library
```

Comment:
Please use `C++` instead of `c++` everywhere in this `.rst` document.

Reply:
Fixed in bb1f1cd.